packages/ui/v2/modules/integrations/DisconnectIntegration.tsx

```diff
@@ -12,11 +12,13 @@ export default function DisconnectIntegration({
   label,
   trashIcon,
   isGlobal,
+  onSuccess,
 }: {
   credentialId: number;
   label: string;
   trashIcon?: boolean;
   isGlobal?: boolean;
+  onSuccess?: () => void;
```

Comment:
Had to add this because in some cases, the disconnect action should for example, redirect the user somewhere else. See https://github.com/calcom/cal.com/pull/4305/files#diff-ca24dc6387663c3bf0b33e52827cb05585264596d8f9b65b02c56565b403dca2R160